### PR TITLE
SS Wheeler 4

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: journal/modernized/37-chapter-34.adoc [hunk 1]
```diff
@@ -671,6 +671,9 @@ will lastingly remain;--that He may be glorified thereby.
 [.offset]
 The following are extracts from the address to the missionaries in New Zealand.
 
+[.embedded-content-document.address]
```
Comment (jaredh159) [inline journal/modernized/37-chapter-34.adoc:674]: nice job flagging this one as an `address` and the last one as `epistle`, perfect. 👍😊

File: journal/modernized/37-chapter-34.adoc [hunk 2]
```diff
@@ -991,11 +994,17 @@ Daniel Wheeler
...
+
 [.offset]
 A copy of a letter from the missionaries at Paihia
 
+[.embedded-content-document.letter]
+--
+
 [.signed-section-context-open]
 Church Missionary Society`'s Station, Paihia, Bay of Islands, New Zealand, 18th January, 1837.
+//Does the linter catch this after i submit the PR? or should it have caught it so I can break up line 1006 now?
```
Comment (jaredh159) [inline journal/modernized/37-chapter-34.adoc]: Yeah, this line is over 90, but under 100, so it's fine.  If you don't see the linter barking about it, it should be fine. I may have never mentioned this, but I fixed all the lines in all the books that were over 100 characters, so the lint rule is really to help not accidentally introduce _new_ long lines while editing. 👍

File: journal/modernized/37-chapter-34.adoc [hunk 3]
```diff
@@ -1394,6 +1414,8 @@ which they have learnt from their exterminating visitors.
 How long we may have to remain here, I know not;
 but I desire to be resigned to wait the blessed Master`'s time.
 
+--
+//I think this is where this letter ends, not certain though.
```
Comment (jaredh159) [inline journal/modernized/37-chapter-34.adoc]: Yup, I think you got it right here.  Maybe you're already doing this, but when you're in doubt, looking it up in the original often helps.  Sometimes in Wheeler it's _still_ not clear to me after looking at the original, but I find it usually helps to see how the original was formatted.

**Review by jaredh159 — CHANGES_REQUESTED** [review]
Maija this one's totally fine -- you just need to go back and remove the comments (you can do a search in the editor for `//`) and re-submit, and I'll merge this one.  Nice work!

File: journal/modernized/37-chapter-34.adoc (hunk 3 above)
Comment (maijapapaya) [reply journal/modernized/37-chapter-34.adoc]: Got it. Sometimes I forget we can look at the originals. :)

**Comment by jaredh159** [comment]
This also looks good, thanks! Merging...

**Pull request merged**